tests/integration/ceph_mgr_test.go

```diff
@@ -0,0 +1,216 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/utils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+// **************************************************
+// *** Mgr operations covered by TestMgrSmokeSuite ***
+//
+// Ceph orchestrator device ls
+// Ceph orchestrator status
+// Ceph orchestrator host ls
+// Ceph orchestrator create OSD
+// Ceph orchestrator ls
+// **************************************************
+func TestMgrSmokeSuite(t *testing.T) {
+	if installer.SkipTestSuite(installer.CephTestSuite) {
+		t.Skip()
+	}
+
+	s := new(MgrSmokeSuite)
+	defer func(s *MgrSmokeSuite) {
+		HandlePanics(recover(), s.op, s.T)
+	}(s)
+	suite.Run(t, s)
+}
+
+type MgrSmokeSuite struct {
+	suite.Suite
+	op        *TestCluster
+	k8sh      *utils.K8sHelper
+	namespace string
+	CLI       MgrCLI
+}
+
+type host struct {
+	Addr     string
+	Hostname string
+	Labels   []string
+	Status   string
+}
+
+type service struct {
+	Container_image_name string
+	Service_name         string
+	Size                 int
+	Running              int
+	Spec                 interface{}
+	Last_refresh         string
+}
+
+func (suite *MgrSmokeSuite) SetupSuite() {
+	suite.namespace = "mgr-ns"
+
+	mgrTestCluster := TestCluster{
+		namespace:        suite.namespace,
+		storeType:        "bluestore",
+		storageClassName: "",
+		useHelm:          false,
+		usePVC:           false,
+		mons:             3,
+		rbdMirrorWorkers: 1,
+		rookCephCleanup:  true,
+		withoutOSDs:      true,
+	}
+
+	suite.op, suite.k8sh = StartTestCluster(suite.T, smokeSuiteMinimalTestVersion, &mgrTestCluster, installer.VersionMaster, installer.MasterVersion)
+
+	suite.CLI = MgrCLI{k8sh: suite.k8sh, namespace: suite.namespace}
+
+}
+
+func (suite *MgrSmokeSuite) AfterTest(suiteName, testName string) {
+	suite.op.installer.CollectOperatorLog(suiteName, testName, installer.SystemNamespace(suite.namespace))
+}
+
+func (suite *MgrSmokeSuite) TearDownSuite() {
+	suite.op.Teardown()
+}
+
+func (suite *MgrSmokeSuite) TestDevicels() {
+	logger.Infof("Testing .... <orch device ls>")
+	err, device_list := suite.CLI.execute([]string{"device", "ls"})
+	assert.Nil(suite.T(), err)
+
+	logger.Infof("device ls RESULT =%v", device_list)
+}
+
+func (suite *MgrSmokeSuite) TestStatus() {
+	logger.Infof("Testing .... <orch status>")
+	err, status := suite.CLI.execute([]string{"status"})
+	assert.Nil(suite.T(), err)
+	assert.Equal(suite.T(), status, "Backend: rook\nAvailable: True")
+}
+
+func (suite *MgrSmokeSuite) TestHostls() {
```

Comment:
Following `go` guidelines, capitalized `Ls`

```suggestion
func (suite *MgrSmokeSuite) TestHostLs() {
```